dotnet/dotnet-guestbook/.readmes/vscode/README.md

```diff
@@ -35,7 +35,10 @@ The Guestbook sample demonstrates how to deploy a Kubernetes application with a
   - `kubernetes-manifests/guestbook-mongodb.deployment.yaml` - deploys a pod containing a MongoDB instance
   - `kubernetes-manifests/guestbook-mongodb.service.yaml` - exposes the MongoDB service on an internal IP in the cluster
 
-  ### Skaffold modules
+### Skaffold modules
+
+>  Note: This feature is currently only available on the [insiders release](https://cloud.google.com/code/docs/vscode/insiders?utm_source=ext&utm_medium=partner&utm_campaign=CDR_kri_gcp_cloudcodereadmes_012521&utm_content=-) of Cloud Code. To get the latest pre-release build of Cloud Code, follow the instructions on [Installing Insiders builds](https://cloud.google.com/code/docs/vscode/insiders#get).
```

Comment:
Please update the url to https://cloud.google.com/code/docs/vscode/insiders.

We don't the text after ?

Reply:
The text after is UTM tracking to get stats on how many users follow the link from the README, it helps track README usage.

Reply:
Got it. Thanks.